src/curves/mnt4_753.nr

```diff
@@ -147,4 +147,11 @@ mod test {
             1,
         );
     }
+
+    // #[test]
+    // fn test_scalar_field_conversion(num: MNT4_753_Fr) {
+    //   let scalar_field = MNT4_753Scalar::from_bignum(num);
+    //    let bignum = scalar_field.into_bignum();
+    //    assert(bignum == num);
+    //}
```

Comment:
`MNT4_753_Fr` is a `[u128; 7]` in the eyes of the fuzzer, but in order to be well formed each element needs to be a valid `u120` (not `u128`!). This means that approx 1 - 0.5**(7*8) of the inputs are invalid.

There's a decent chance that this test can be fixed by generating the bignum through `MNT4_753_Fr::derive_from_seed`